adapters/repos/db/vector/hnsw/persistence_integration_test.go

```diff
@@ -53,12 +53,12 @@ func TestHnswPersistence(t *testing.T) {
 	require.Nil(t, err)
 
 	for i, vec := range testVectors {
-		err := index.Add(i, vec)
+		err := index.Add(int64(i), vec)
```

Comment:
whole file: see above, all node ids should be represented by `uint64`, removing the need for the conversions.